tripy/tripy/frontend/shape.py

```diff
@@ -24,6 +24,62 @@
 import tripy.frontend.utils as frontend_utils
 
 
+@export.public_api()
+class ScalarShape(Tensor):
```

Comment:
Maybe `ShapeScalar` is more accurate? It is a scalar that came from a shape, not a shape that is scalar.